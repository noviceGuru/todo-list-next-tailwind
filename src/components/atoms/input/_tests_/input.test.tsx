import { render, screen } from "@testing-library/react"

import Input from "../input"

it('renders Input with the correct text and looks', () => {
	render(<Input
		onChange={()=>{}}
		defaultValue="default dummy text for input"
	/>)
	
	const input = screen.getByRole('textbox')

	expect((input as HTMLInputElement).value).toBe("default dummy text for input")
	expect(input).toMatchSnapshot()
})