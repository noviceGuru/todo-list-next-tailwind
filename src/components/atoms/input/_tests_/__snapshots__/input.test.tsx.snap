// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Input with the correct text and looks 1`] = `
<input
  class="border-solid border-2 border-orange-500 rounded"
  value="default dummy text for input"
/>
`;
